### PR TITLE
Feature/update form to reflect latest changes

### LINKED ISSUE #24: Update form to reflect latest changes
The radio group is now being replaced with nested form fields. These fields will include a select and text fields. See doc

### PR DESCRIPTION
**Pull request by dlouise64**
# What has changed and why?
Added new form components
Styles changed
Code tidy

This is going to be merged in to address lookup as I branch off of there.
# How to test this change

- [ ] has automated tests
- [x] at least one a reviewer ran the code
- [x] tested in small screen
- [ ] i18n

closes #24

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,7 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 .sass-cache
+
+/src/styles/App.css.map
```
Comment (Br3nda) [inline .gitignore]: Don't think you need this line if you have *.css.map below it.

File: .travis.yml [hunk 2]
```diff
@@ -12,5 +12,4 @@ deploy:
   skip-cleanup: true
   github-token: $GITHUB_TOKEN
   keep-history: true
-  on:
-    branch: master
```
Comment (Br3nda) [inline .travis.yml]: If we remove the `on: branch: master`, then travis will try to deploy on every branch.

File: src/components/FirstTimeApplicant/Page1.js [hunk 3]
```diff
@@ -5,18 +5,21 @@ import validate from '../../helpers/validate';
 import { scrollToFirstError } from '../../components/Forms/FormScroll';
 import LanguageToggle from '../../components/Forms/LanguageToggle';
 import { underscorize, camelCaser } from '../../helpers/strings';
+// import { RenderCheckbox } from '../Forms/RenderCheckbox';
```
Comment (Br3nda) [inline src/components/FirstTimeApplicant/Page1.js]: Can we remove this line?

File: src/components/FirstTimeApplicant/Page1.js [hunk 4]
```diff
@@ -34,38 +37,114 @@ class SelectingFormValuesForm extends React.Component {
...
   render() {
+    let shown = {
+			display: this.state.shown ? "block" : "none"
+    };
     let lang = this.state.lng;
     return (
       <Fragment>
         <LanguageToggle handler={this.handleLanguageChange} langState={this.state.lng} />
-        <form className="container" onSubmit={this.props.handleSubmit(values => console.log(values))}>
+        {/* <form className="container" onSubmit={this.props.handleSubmit(values => console.log(values))}> */}
```
Comment (Br3nda) [inline src/components/FirstTimeApplicant/Page1.js]: Can you remove the commented out bit?

File: src/components/FirstTimeApplicant/Page1.js [hunk 5]
```diff
@@ -78,24 +157,28 @@ SelectingFormValuesForm = reduxForm({
...
 SelectingFormValuesForm = connect(state => {
-  const hasSuperAnnuationValue = selector(state, 'hasSuperAnnuationChecked');
-  const hasWageOrSalaryValue = selector(state, 'hasWageOrSalaryChecked');
-  const hasLivedHereValue = selector(state, 'hasLivedHereChecked');
-  const hasJobSeekerBenefitValue = selector(state, 'hasJobSeekerBenefitChecked');
-  const hasOtherValue = selector(state, 'hasOtherChecked');
-  const hasPartnerValue = selector(state, 'hasPartnerChecked');
-  const hasSupportedLivingValue = selector(state, 'hasSupportedLivingChecked');
-  const hasSelfEmployedValue = selector(state, 'hasSelfEmployedChecked');
+//   const hasSuperAnnuationValue = selector(state, 'hasSuperAnnuationChecked');
```
Comment (Br3nda) [inline src/components/FirstTimeApplicant/Page1.js]: is this being saved for something?

**Comment by Br3nda** [comment]
The address lookup is not working anymore.

**Comment by Br3nda** [comment]
Are we doing the income widget in a later story? It's a bit confusing like this.
It seems to be asking me where my superannuation comes from. I guess I write "New Zealand"
image

**Comment by Br3nda** [comment]
The address lookup is working again.

**Comment by Br3nda** [comment]
This PR is into `feature/address_lookup`.
Maybe it should be into master.

**Comment by dlouise64** [comment]
@Br3nda yeah the income is another ticket which the UI has been updated recently.

File: src/components/Forms/RadioWithRadio.js [hunk 6]
```diff
@@ -66,8 +66,8 @@ export default class RadioWithRadio extends React.Component {
               <fieldset  style={{marginTop: '35px'}}>
                 <legend style={{marginBottom: '15px'}}>{this.props.optionsText[1]}</legend>
                 <div>
-                  {['Yes', 'No'].map(item=>{
-                    return <label><input type="radio" name="test1" onClick={this.sub.bind(this)} /><span>{item}</span></label>;
+                  {['Yes', 'No'].map((item, key)=>{
+                    return <label key={key}><input type="radio" name="test1" onClick={this.sub.bind(this)} /><span>{item}</span></label>;
```
Comment (Br3nda) [inline src/components/Forms/RadioWithRadio.js]: :sparkles:

**Pull request merged**